### PR TITLE
Cleanup Metric aggregators and unused code

### PR DESCRIPTION
**Pull request by cijothomas**
Fixes #.

## Changes

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [ ] `CHANGELOG.md` updated for non-trivial changes
* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

File: src/OpenTelemetry/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Metrics perf improvements, bug fixes.
+  Replace MetricProcessor with Metric reader.
```
Comment (reyang) [inline src/OpenTelemetry/CHANGELOG.md:6]:
```suggestion
  Replace MetricProcessor with MetricReader.
```

**Review by reyang — APPROVED** [review]

File: src/OpenTelemetry/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Metrics perf improvements, bug fixes.
+  Replace MetricProcessor with MetricReader.
```
Comment (reyang) [inline src/OpenTelemetry/CHANGELOG.md:6]: Add the PR links.

**Comment by mic-max** [comment]
Did you run some tool to find the unused code? If not, that could be an idea to automate the burial of the dead code. Thoughts?

**Comment by reyang** [comment]
> Did you run some tool to find the unused code? If not, that could be an idea to automate the burial of the dead code. Thoughts?

I guess no. "Unused code" is hard to define, especially for libraries.

**Pull request merged**